ballerina/Module.md

```diff
@@ -1,14 +1,110 @@
 ## Overview
 
-[//]: # (TODO: Add overview mentioning the purpose of the module, supported REST API versions, and other high-level details.)
+[Jira](https://www.atlassian.com/software/jira)  is a widely-used issue and project tracking software by Atlassian. It provides a REST API that allows applications to connect with Jira to create, update, and delete various resources, enabling efficient project and task management.
+
+The `ballerinax/jira` package offers APIs to interact with the [Jira REST API](https://developer.atlassian.com/cloud/jira/platform/rest/v3/intro), providing access to various endpoints for managing issues, filters, projects, and other resources.
 
 ## Setup guide
 
-[//]: # (TODO: Add detailed steps to obtain credentials and configure the module.)
+To use the Jira connector, you must have a Jira Cloud account with API access. If you do not have one, you can sign up for a Jira account on [Atlassian’s website](https://developer.atlassian.com/cloud/jira/platform/).
+
+### Step 1: Create an API Token
+1. Login to [Atlassian Account](https://developer.atlassian.com/cloud/jira/platform/)
+
+2. Click on the profile icon to get into developer console
+
+![{D8EEDDDF-5D02-4907-A10A-911BB513050F}](https://github.com/user-attachments/assets/3ec0f5be-6c5f-4038-a37a-ec8a2f6e2ccf)
```

Comment:
https://github.com/ballerina-platform/module-ballerinax-jira/pull/5/files#r1836165271

Reply:
Done

Reply:
I guess this is not done. Could you please do the same as you did in the Readme file?